src/Aspire.Hosting/Dashboard/proto/resource_service.proto

```diff
@@ -127,8 +133,12 @@ message Resource {
     optional string state = 5;
     optional google.protobuf.Timestamp created_at = 6;
     repeated EnvironmentVariable environment = 7;
+    // Deprecated
     optional int32 expected_endpoints_count = 8;
```

Comment:
https://protobuf.dev/programming-guides/proto3/#options

You can add deprecated option to a field. To be honest I'm not exactly sure what impact it has in C# (it might add obsolete attribute, or it might do nothing). But give it a try :smile:

e.g
```proto
optional int32 expected_endpoints_count = 8 [deprecated = true];
```

Reply:
Done